### PR TITLE
Remove inaccurate references to "garbage", fix default for Orleans 7.x+

### LINKED ISSUE #39451: Orleans - Default Collection Age Limit is incorrect
### Type of issue

Outdated article

### Description

The default collection age for Orleans is now 15 minutes. This page states it is 2 hours, which was the default in v3. There doesn't appear to be any version selector, so I assume this page is relevant to the latest version of Orleans.

@ReubenBond

### Page URL

https://learn.microsoft.com/en-us/dotnet/orleans/host/configuration-guide/activation-garbage-collection

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/orleans/host/configuration-guide/activation-garbage-collection.md

### Document Version Independent Id

c78c9d33-f923-6ec5-d649-9a09630df68f

### Article author

@IEvangelist

### Metadata

* ID: 4734308a-e5ca-67e8-9090-f7118eb8289b 
* Service: **dotnet-orleans**

---
[Associated WorkItem - 209076](https://dev.azure.com/msft-skilling/Content/_workitems/edit/209076)

### PR DESCRIPTION
**Pull request by ReubenBond**
Fixes #39451

Also, use the term "activation collection" consistently and drop the term "garbage" to hopefully reduce conflation with GC.

@IEvangelist is there a way to change the URL to activation-collection instead of activation-garbage-collection and redirect from the latter to the former?

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/orleans/host/configuration-guide/activation-collection.md](https://github.com/dotnet/docs/blob/e224de02ab3e0ae4d3db71976cbef5b358368910/docs/orleans/host/configuration-guide/activation-collection.md) | [Activation collection](https://review.learn.microsoft.com/en-us/dotnet/orleans/host/configuration-guide/activation-collection?branch=pr-en-us-39452) |

### CODE REVIEW

**Comment by IEvangelist** [comment]
Hey @ReubenBond - yeah, you can rename the file, and update all references to the file, i.e.; any TOC.yml and index.yml/md files, or other .md files that link to it. Then you'd add a redirect entry in the orleans redirection file at the root _/docs_ dir of the repo. Happy to help with that, does it make sense?

**Comment by ReubenBond** [comment]
@IEvangelist ptal

File: docs/orleans/host/configuration-guide/activation-collection.md [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
 This article applies to: ✔️ Orleans 3.x and earlier versions
+:::zone-end
 
 A *grain activation* is an in-memory instance of a grain class that gets automatically created by the Orleans runtime on an as-needed basis as a temporary physical embodiment of a grain.
 
-Activation Garbage Collection (Activation GC) is the process of removal from memory of unused grain activations. It's conceptually similar to how garbage collection of memory works in .NET. However, Activation GC only takes into consideration how long a particular grain activation has been idle. Memory usage isn't used as a factor.
+Activation Collection is the process of removal from memory of unused grain activations. It's conceptually similar to how garbage collection of memory works in .NET. However, Activation Collection only takes into consideration how long a particular grain activation has been idle. Memory usage isn't used as a factor.
 
-## How activation GC works
+## How activation collection works
```
Comment (gewarren) [inline docs/orleans/host/configuration-guide/activation-collection.md:25]: Activation collection is lower case here but upper case in other places. We should be consistent - I'd suggest lower case.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**